website/docs: Update ArgoCD integration docs. (#9684)
goauthentik/authentik: website/integrations/services/argocd/index.md

-   Name: ArgoCD
-   Provider: ArgoCD
-   Slug: argocd
-   Launch URL: https://argocd.company/auth/login

### Step 3 - ArgoCD Group creation

Create a new _Group_ (under _Directory/Groups_) that'll be used as the admin group for ArgoCD (if you already have an "admin" group, you can skip this part!)

-   Name: ArgoCD Admins
-   Members: Add your user and/or any user that should be an ArgoCD admin

You can create another group for read-only access to ArgoCD as well if desired:

-   Name: ArgoCD Viewers
-   Members: Any user that should have ArgoCD read-only access

## Terraform provider

```hcl
data "authentik_flow" "default-provider-authorization-implicit-consent" {
  slug = "default-provider-authorization-implicit-consent"
}

data "authentik_scope_mapping" "scope-email" {

>   name = "authentik default OAuth Mapping: OpenID 'email'"

Author: Clean! I didn't realize there was a data source for these. I can make that change. I can make it on the Grafana one as well in this same PR if desired as I lifted this method from there.

>   name = "authentik default OAuth Mapping: OpenID 'email'"

Reviewer: "goauthentik.io/providers/oauth2/scope-profile",

dex.authentik.clientSecret: <base 64 encoded value of the Client Secret from the Provider above>
```

> If using helm, the above can be added to `configs.secret.extra` like so, securely substituting the string however you see fit:

Reviewer 2: ```suggestion
If using Helm, the above can be added to `configs.secret.extra` as shown below, securely substituting the string however you see fit:
```

Reviewer 2: I'd like to further edit this, to identify/qualify and be very specific on what "the above <?>" is... Can we say 

"If using Helm, the above string/line/definition/? can be added to a  `configs.secret.extra` section in the <__??__> file, as shown below, securely substituting the string however you see fit:"

These are nits (well not the capitalization, that's important), but for new users, it's always nice to be a bit pedantic, and say where (in what file) we are talking about.

Author: Ammended, see what you think!
